packages/select-rich/src/LionSelectRich.js

```diff
@@ -159,6 +159,12 @@ export class LionSelectRich extends ScopedElementsMixin(
     return this.modelValue;
   }
 
+  // Duplicating from FormGroupMixin, because you cannot independently inherit/override getter + setter.
+  // If you override one, gotta override the other, they go in pairs.
+  set serializedValue(value) {
+    this._setCheckedElements(value, (el, val) => el.serializedValue.value === val);
```

Comment:
I think this should work... e.g. we have it but we still call the parent one

```suggestion
    super.serializedValue = value;
```

Reply:
Good one, works!